.doc_gen/metadata/iam_metadata.yaml

```diff
@@ -456,6 +485,18 @@ iam_DeleteAccessKey:
   synopsis: delete an &IAM; access key.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/IAM
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.DeleteAccessKeyExample
+            - description:
+              snippet_tags:
+                - IAM.dotnetv3.DeleteResourcesAsync
```

Comment:
These access key part of these two examples is almost identical. Include only one--I'd pick the second because it's the tested one, but it's your call.

Reply:
again, removed the first one